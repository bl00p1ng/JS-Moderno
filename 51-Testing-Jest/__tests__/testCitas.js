import Citas from '../js/classes/Citas.js';

describe('Probar la clase de Citas', () => {
    test('Agregar una nueva cita', () => {
        const citas = new Citas();

        const citaObj = {
            mascota: '',
            propietario: '',
            telefono: '',
            fecha: '',
            hora:'',
            sintomas: ''
        }

        citaObj.id = Date.now();

        citas.agregarCita(citaObj);

        expect(citas).toMatchSnapshot();
    });
});