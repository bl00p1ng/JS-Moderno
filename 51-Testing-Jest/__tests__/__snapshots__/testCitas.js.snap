// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de Citas Agregar una nueva cita 1`] = `
Citas {
  "citas": Array [
    Object {
      "fecha": "",
      "hora": "",
      "id": 1657136880945,
      "mascota": "",
      "propietario": "",
      "sintomas": "",
      "telefono": "",
    },
  ],
}
`;
